ADR/0041-integration-service-upstreaming.md

```diff
@@ -0,0 +1,95 @@
+# XX. Creating Artifacts from Snapshots for testing and release
+
+Date: 2024-10-21
+
+## Status
+
+Proposed
+
+## Authors
+
+- Krunoslav Pavic
+- Andrew McNamara
+
+## Context
+
+This ADR aims to cover the proposed functionality for creating additional Artifacts as part of the Konflux build/test/release process.
+These artifacts would vary in type and would cover use cases where either the single built image or entire sets of images
+from the user's application are needed to generate one or more artifacts.
+
+## Use Cases in Detail
```

Comment:
this section describes the current functionality or the one we will seek for?

Reply:
It describes the use cases that we want to cater  to.